src/applications/claims-status/containers/FilesPage.jsx

```diff
@@ -85,6 +87,12 @@ class FilesPage extends React.Component {
         <ClaimFileHeader isOpen={isOpen} />
         <Toggler toggleName={Toggler.TOGGLE_NAMES.cstShowDocumentUploadStatus}>
           <Toggler.Enabled>
+            <AdditionalEvidencePage additionalEvidenceTitle="Upload additional evidence" />
+            <div className="vads-u-margin-y--6 vads-u-border--1px vads-u-border-color--gray-light" />
```

Comment:
Do we know if the design system has something that creates these lines? If not, this is great!